### PR TITLE
1st negative number in every window size of k

### PR DESCRIPTION
**Pull request by bitaashna**
#2304
Description

One of the fixed window problem has been solved and implemented using c++.

Checklist

I've read the contribution guidelines.
I've checked the issue list before deciding what to submit.
I've edited the README.md and link to my code.
#C++
#2304

### CODE REVIEW

**Comment by bitaashna** [comment]
@siddharth25pandey Now also facing the same issue.
One of the checks is failing.
DeepSource: Java (beta)
what should i do?

**Comment by siddharth25pandey** [comment]
> @siddharth25pandey Now also facing the same issue.
> One of the checks is failing.
> DeepSource: Java (beta)
> what should i do?

@bitaashna Ignore it, no need to worry about as Deepsource Presently don't supports C++

File: C-Plus-Plus/README.md [hunk 1]
```diff
@@ -236,6 +236,16 @@ _add list here_
...
 
+## Sliding Window
+
+- [Maximum sum subarray of size k](sliding_window/max_sum_subarray_of_size_k.cpp)
+- [1st negative number in every window size of k](sliding_window/1st_negative_no_in_every_window_of_size_k.cpp)
+- [Count occurence of Anagram](sliding_window/count_occurences_of_anagrams.cpp)
+- [Maximum of all subarray of size k](sliding_window/maximum_of_all_subarray_of_size_k.cpp)
+- [Largest subarray of sum k](sliding_window/largest_subarray_of_sum_k.cpp)
+- [largest substring with k distinct character](sliding_window/longest_substring_with_k_unique_characters.cpp)
+- [length of largest substring with no repeating characters](sliding_window/length_of_longest_substring_with_no_repeating_characters.cpp)
```
Comment (siddharth25pandey) [inline C-Plus-Plus/README.md:247]: Remove this

File: C-Plus-Plus/README.md [hunk 2]
```diff
@@ -236,6 +236,16 @@ _add list here_
 - [Minimum number of insertions and deletions](dp/min_ins_del.cpp)
 - [Unbounded Knapsack](dp/unbounded_knapsack.cpp)
 
+## Sliding Window
+
+- [Maximum sum subarray of size k](sliding_window/max_sum_subarray_of_size_k.cpp)
```
Comment (siddharth25pandey) [inline C-Plus-Plus/README.md:241]: Remove this

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@bitaashna as told you earlier, please create one PR per program
* Remove the count_of_occurences_of_anagrams file

File: C-Plus-Plus/README.md (hunk 1 above)
Comment (bitaashna) [reply C-Plus-Plus/README.md:247]: I have removed the file.
But in readme I have updated all at once.
Should I do it again?
Please let me know

**Comment by bitaashna** [comment]
@siddharth25pandey  I have updated the readme file at once but i have added files one by one by making different branches in forked repository.
Screenshot from 2021-03-15 17-55-04
To uncommit the changes I deleted my repo :( from main3 branch. Now I am not able to commit it again.Please help me regarding this.

**Comment by siddharth25pandey** [comment]
@bitaashna Duplicate [PR](https://github.com/TesseractCoding/NeoAlgo/pull/3277)

**Pull request closed without merging**